### ISSUE TITLE
is paddle inner loss computed is output_loss?

### ISSUE THREAD

**Issue by youngstu** [PaddlePaddle/Paddle#9649]
```
    # class loss
    class_loss = classification_cost(
      input=cls_score, label=cls_label)

    # reg loss
    reg_loss = smooth_l1_cost(
      input=reg_pos, label=reg_label)

    if maxid != cls_label:
      reg_loss = 0

    output_loss.append(class_loss)
    output_loss.append(reg_loss)

    inputs(img, class_label, bbox_label)
    outputs(output_loss)
```

**Comment by youngstu** [comment]
我怎样自己写一个cost 函数，里面根据条件调用不同loss 函数？可以大概提供一个思路？

**Comment by youngstu** [comment]
需要修改底层C++实现代码，还是在python 接口层就可以做到？

**Comment by qingqing01** [comment]
> 如果使用只是 loss 定义层使用 Fluid Api, 其他使用 python Api v1 or v2.

这两个不可混合使用~
